### ISSUE TITLE
after episode is finished, it does not jump back to menu

### ISSUE THREAD

**Issue by jameskimmel** [jellyfin/jellyfin-media-player#634]
**Describe the bug**
Since updating to v1.10.0 and 10.9.1, with autoplay disabled, when a episode is finished, the player does not jump back to the menu anymore. Instead it will stay at a black screen. 

**To Reproduce**
1. disable autoplay
2. play an episode until the end
3. screen will stay black

**Expected behavior**
Jump back to the menu like it did before. 

**Desktop (please complete the following information):**
 - OS: macOS 
 - Version: 14.5
 - Installation Method: silicon dmg

**Comment by photoenix** [comment]
Bug still present in v1.10.1 and 10.9.2 on Windows.
Additionally: after clicking on Back, I am no longer able to scroll. Scrollbar is gone, mouse wheel doesn't work, neither do up and down arrows or page up and down. So after I watch an episode I am pretty much forced to restart the player :/

**Comment by jameskimmel** [comment]
Still present in v10.9.2 and v1.10.1

**Comment by titustide** [comment]
Still present in v10.9.3, I also cannot scroll afterwards. After pressing the back button to the menu, the backdrop is also gone.

**Comment by marksugarman** [comment]
Still present in v10.9.4 on Windows 11

**Comment by mihailim** [comment]
For what it's worth, both behaviors (player black screen after finishing a video, _and_ inability to scroll) are present in v1.11.1 Linux flatpak, with jellyfin-web 10.9.7 from the official repos.

**Comment by craigers521** [comment]
also on the linux flatpak.  One of the biggest issues with this bug for me is that i use my computer as a HTPC, and often fall alseep to watching some tv... i  disable autoplay so it doesnt just play continuously, but if the player doesnt return to menu after an episode then the computer just thinks its actively playing content, so it will never go into sleep/suspend and just keep my computer and monitors on all night.

**Comment by craigers521** [comment]
@iwalton3 i will pay you to fix this bug. name your price its driving me insane.

**Comment by grant841** [comment]
@craigers521 Dude seriously I am right there with you. I just experienced a different variation of the bug, when pressing back from the player view and while in full screen mode, it switches the entire application to my other monitor even if it is off. Only happened a few times though.

**Comment by dmitrylyzo** [comment]
Since 10.9, this is intended behaviour (not returning to the menu): https://github.com/jellyfin/jellyfin-web/pull/5398
The only thing, it usually displays a backdrop instead of black screen.

**Comment by craigers521** [comment]
> Since 10.9, this is intended behaviour (not returning to the menu): [jellyfin/jellyfin-web#5398](https://github.com/jellyfin/jellyfin-web/pull/5398) The only thing, it usually displays a backdrop instead of black screen.

its intended for the player to sit on a black screen indefinitely preventing suspend, and then if you hit back button scroll functionality is broken?

**Comment by grant841** [comment]
> Since 10.9, this is intended behaviour (not returning to the menu): [jellyfin/jellyfin-web#5398](https://github.com/jellyfin/jellyfin-web/pull/5398) The only thing, it usually displays a backdrop instead of black screen.

You misunderstand, you are correct in that it is not supposed to do anything when the "play next episode automatically" setting is set to off.  The problem is that sometimes once you get to that point, it disallows you to go back to main menu when clicking the back button, sometimes it goes back to menu and there is no backdrop or scrolling, sometimes it switches the application to another monitor when pressing back button, and sometimes it doesn't play the next episode automatically even when the setting is set to on. None of which are the intended behavior. This is my experience on WIN10, almost 100% of the time at least one of these bugs occur for me and I am slowly going insane send help

**Comment by craigers521** [comment]
i rolled back my flatpak to an earlier commit.  I've lost a few QoL things but worth it to fix this weird autoplay behavior.

**Comment by craigers521** [comment]
@grant841 didnt touch server, just JMP.  I'll have to check the specific commit when i get back to my computer but it was one of the ones before 1.10 release

**Comment by grant841** [comment]
> @grant841 didnt touch server, just JMP. I'll have to check the specific commit when i get back to my computer but it was one of the ones before 1.10 release

yes please let me know as soon as possible, I don't remember which version started to have the issue

**Comment by craigers521** [comment]
@grant841 
![image](https://github.com/user-attachments/assets/65da541a-fcf9-469a-b8b3-057f16d7b46b)
if you are using the flatpak its commit 6299cda1c289afc47fd2cb4413eba9dfc2e033c946e9e9a217415de5601ecefd.  which i think is version 1.9.1.

**Comment by grant841** [comment]
@craigers521 no I am on WIN10 but I will try to install that version and see if it helps, thanks!

**Comment by slackerbob** [comment]
I can also confirm all of this using the flatpak on ArchLinux and Fedora Silverblue.

Not being able to scroll any window until I close and re-open the application -every-single-time- is frustrating to no end.

**Comment by titustide** [comment]
> [@grant841](https://github.com/grant841) ![image](https://github.com/user-attachments/assets/65da541a-fcf9-469a-b8b3-057f16d7b46b) if you are using the flatpak its commit 6299cda1c289afc47fd2cb4413eba9dfc2e033c946e9e9a217415de5601ecefd. which i think is version 1.9.1.

You had to uninstall your current version before installing the earlier one correct? How do I retain all of my data when doing this?

**Comment by mihailim** [comment]
For what it's worth, this bug is still present in the Linux flatpak build `1.12.0` at commit `9b8944642950247fb11536e01846de5b1fcfdb94411214c9bfb724afc4eddd18`, including the behavior where scrolling on any screen is broken once navigating back from the end of a playback session when server-side autoplay is off.

**Comment by brokenshield** [comment]
Same for me on flatpack version of JMP version 10.10.7

Thank you for all the development of Jellyfin. It's a great system and I really like it.

**Comment by somehuman60** [comment]
> For what it's worth, this bug is still present in the Linux flatpak build `1.12.0` at commit `9b8944642950247fb11536e01846de5b1fcfdb94411214c9bfb724afc4eddd18`, including the behavior where scrolling on any screen is broken once navigating back from the end of a playback session when server-side autoplay is off.

Same here on Linux Mint,

**Comment by WallK** [comment]
Specifically the scrolling bug is very annoying and still is happening on win11, for example

**Comment by dimmas375** [comment]
Black screen preventing suspend is present on Debian 13.1 KDE and Gnome, and recent KDE Neon. It's quite strange when tablet/HTPC/Laptop don't go sleep after the playback is stopped and there aren't any other active tasks.

**Comment by Noradmin50** [comment]
Same here with server 10.11.3 (docker) and player 1.12.0 (win 10).
This bug must not be very widespread because no one is talking about it, but it would be good to look into it because it's a bit annoying.

**Comment by Kory-Albert** [comment]
I'm not sure why, but this stopped occurring for me using the same player version as @Noradmin50.

Jellyfin 10.10.7 container (kubernetes)
JMP 1.12.0 flatpak fedora 42 silverblue

**Comment by tehmasterer** [comment]
Still an issue in 2026 using Jellyfin 10.11.6 (docker) and the latest Windows 11 desktop app (v1.12.0 - Bugfix Release).

If you have autoplay disabled, the episode ends on a black screen, instead of ending playback. Upon pressing the back arrow, the scrollbar disappears, and the client is basically non-responsive, forcing you to close it.

**Comment by mihailim** [comment]
Can confirm, this still happens with server-side 10.11.6 and the Flatpak Jellyfin Desktop 2.0.0 at commit `101468e6a77cf4cc4343f3069933025a86ebaf765eb662644dc6536003831c1d` .

**Comment by Noradmin50** [comment]
I thought I noticed that this doesn't happen when you read the last episode of the series, but I'm not sure.
If that helps...

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
